### PR TITLE
stars solution

### PR DESCRIPTION
**Pull request by AlNeon**
- [DEMO LINK](https://AlNeon.github.io/layout_stars/)
- [TEST REPORT LINK](https://AlNeon.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,49 @@
...
-    <h1>Stars</h1>
+    <section>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
```
Comment (Viktor-Kost) [inline src/index.html:19]: add an empty line between `multiline` siblings for better readability
```suggestion
      </div>
      
      <div class="stars stars--1">
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```
Comment (Viktor-Kost) [inline src/style.css:14]: you can use background `shorthand`

**Review by Viktor-Kost — APPROVED** [review]
Great!
Just left some comments for you to know

**Comment by AlNeon** [comment]
added readme.md & some improvments v.2

**Pull request closed without merging**